packages/core/admin/ee/server/services/review-workflows/workflows.js

```diff
@@ -11,8 +14,18 @@ module.exports = ({ strapi }) => ({
     return strapi.entityService.findOne(WORKFLOW_MODEL_UID, id, opts);
   },
 
-  create(workflowData) {
-    return strapi.entityService.create(WORKFLOW_MODEL_UID, { data: workflowData });
+  async create(opts) {
+    if (!opts.data.stages) {
+      throw ApplicationError('Can not create a workflow without stages');
```

Comment:
I think you need `new ApplicationError` here

Reply:
I don't want to be picky, but since this violates a constraint should this be a ValidationError instead?